### PR TITLE
Update ubuntu version

### PR DESCRIPTION
**Pull request by Forgind**

### CODE REVIEW

File: .vsts-dotnet-ci.yml [hunk 1]
```diff
@@ -147,7 +147,7 @@ jobs:
 - job: CoreBootstrappedOnLinux
   displayName: "Linux Core"
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```
Comment (rainersigwald) [inline .vsts-dotnet-ci.yml:150]: Should we peg to `20.04` instead of `latest`? more manual updates later but more explicit for servicing branches.

**Review by rainersigwald — APPROVED** [review]

File: .vsts-dotnet-ci.yml (hunk 1 above)
Comment (BenVillalobos) [inline .vsts-dotnet-ci.yml:150]: While we're here, I wonder if the mac os vmimage has an equivalent `macOS-10.14` to `macOS-latest`. Even if it does, I'd prefer CI PR's to be hyper-specific

**Review by BenVillalobos — APPROVED** [review]

File: .vsts-dotnet-ci.yml (hunk 1 above)
Comment (rainersigwald) [reply .vsts-dotnet-ci.yml:150]: The answer to your question is here: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

Comment (BenVillalobos) [reply .vsts-dotnet-ci.yml:150]: I vote on latest (less steps for the team)

> but more explicit for servicing branches.

has this ever been relevant for linux?

Comment (rainersigwald) [reply .vsts-dotnet-ci.yml:150]: Not that I can think of; it's more relevant in SDK/installer land.

**Pull request merged**